package/com.unity.formats.usd/Tests/Editor/SceneTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿using NUnit.Framework;
```

Comment:
Thanks for adding the test @lucillecaillaud That's what I had in mind.
But thinking about it, this test should really go with the other USD.NET tests. I'll move it there.